packages/karma-esm/TODO.md

```diff
@@ -0,0 +1,2 @@
+- don't run babel on mocha etc. it's now passed to es dev server
```

Comment:
we probably want to remove this file at some point right?

Reply:
Yeah that file is no longer needed! I've removed